### PR TITLE
Bugfix: Dexcom driver should allow for no calibration records

### PR DESCRIPTION
**Pull request by gniezen**
See https://trello.com/c/tlpAbIxw for details.

Ping @jebeck for review and testing.

### CODE REVIEW

**Comment by gniezen** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;tlpAbIxw&#x2F;8-ux-0-dev-2-tst-1-uploader-dexcom-g5-receiver-with-no-calibrations-gets-stuck-on-empty-meter-data-partition-during-upload"> &lt;ux:0 dev:2 tst:1&gt; [uploader] [Dexcom] G5 receiver with no calibrations gets stuck on empty METER_DATA partition during upload</a>

File: lib/drivers/dexcomDriver.js [hunk 1]
```diff
@@ -1003,6 +1008,8 @@ module.exports = function (config) {
                 progress(100);
                 cb(err, data);
               });
+            } else {
+              cb(err, data);
```
Comment (jebeck) [inline lib/drivers/dexcomDriver.js]: Was the lack of this the regression re: the error bubbling up to be caught by the UI?

Comment (gniezen) [inline lib/drivers/dexcomDriver.js]: Yes, so it wasn't really a regression - the error just wasn't being passed up the chain in this specific instance (no meter data).

**Comment by jebeck** [comment]
Works like a dream (\o/), code LGTM. :shipit:

**Comment by gniezen** [comment]
Going to merge this, to be included in the next release as it's not considered high priority at the moment.

**Pull request merged**